boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -287,6 +287,7 @@ WinLdrLoadImage(IN PCHAR FileName,
     Status = ArcOpen(FileName, OpenReadOnly, &FileId);
     if (Status != ESUCCESS)
     {
+        WARN("Error while opening %s, Status: %u\n", FileName, Status);
```

Comment:
What about putting the string identifier between single quotes? (this would allow seeing where the string ends & starts, in case some invalid buffer with whitespace is provided.)
+ Same "suggestion" for all the other added traces.